synapse/storage/databases/main/schema/delta/59/06chain_cover_index.sql

```diff
@@ -0,0 +1,17 @@
+/* Copyright 2020 The Matrix.org Foundation C.I.C
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+INSERT INTO background_updates (ordering, update_name, progress_json, depends_on) VALUES
+  (5829, 'chain_cover', '{}', 'rejected_events_metadata');
```

Comment:
Annoyingly this is now `5906`....

```suggestion
  (5906, 'chain_cover', '{}', 'rejected_events_metadata');
```

Reply:
:facepalm: